provisioning/ci-setup-cluster.sh

```diff
@@ -28,6 +28,8 @@ export DATABASE_URL_BASE64
 export DATABASE_PASSWORD_BASE64
 export TEST_AWS_SECRET_ACCESS_KEY_BASE64
 
+export MESSENGER_TRANSPORT_DSN="${AWS_SQS_QUEUE_URL}?auto_setup=false"
```

Comment:
nemělo by tu být místo AWS_SQS_QUEUE_URL -> SQSQueueUrl? podle https://github.com/keboola/notification-api-php-client/pull/20/files#diff-e0af8aa604e7a7ea7a00ce15bafa785a48832ed9b19923f646a47e8ff55a410fR70

Reply:
My mame envy zvyklost pojmenovavat takhle. Jako mohl bych to jeste prejmenovat v ty CF template no :) to udelam

Reply:
Tak neudelam

> Template format error: Outputs name 'AWS_SQS_QUEUE_URL' is non alphanumeric.
Takze je to imho takhle ok. Navic do budoucna to za tebe vyresi vsechno TF